src/patternfly/components/Alert/alert.scss

```diff
@@ -3,215 +3,166 @@
-  --pf-c-alert--m-inline--before--Bottom: calc(-1 * var(--pf-c-alert--m-inline--BorderBottomWidth));
-  --pf-c-alert--m-inline--before--BackgroundColor: var(--pf-global--default-color--200);
-  --pf-c-alert--m-inline__icon--FontSize: var(--pf-global--icon--FontSize--md);
-  --pf-c-alert--m-inline__icon--Color: var(--pf-global--default-color--200);
-  --pf-c-alert--m-inline__icon--BackgroundColor: transparent;
-  --pf-c-alert--m-inline__icon--PaddingTop: calc(var(--pf-c-alert__icon--Padding) + ((var(--pf-c-alert__icon--FontSize) - var(--pf-c-alert--m-inline__icon--FontSize)) / 2));
-  --pf-c-alert--m-inline__icon--PaddingRight: 0;
-  --pf-c-alert--m-inline__icon--PaddingBottom: var(--pf-global--spacer--md);
-  --pf-c-alert--m-inline__icon--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-alert--m-inline--m-warning__icon--FontSize: #{pf-font-prem(17px)}; // Apply a custom size to the FA icon to ensure all icons are the same width
-  --pf-c-alert--m-inline--m-success__icon--Color: var(--pf-global--success-color--100);
-  --pf-c-alert--m-inline--m-success--before--BackgroundColor: var(--pf-global--success-color--100);
-  --pf-c-alert--m-inline--m-danger__icon--Color: var(--pf-global--danger-color--100);
-  --pf-c-alert--m-inline--m-danger--before--BackgroundColor: var(--pf-global--danger-color--100);
-  --pf-c-alert--m-inline--m-warning__icon--Color: var(--pf-global--warning-color--100);
-  --pf-c-alert--m-inline--m-warning--before--BackgroundColor: var(--pf-global--warning-color--100);
-  --pf-c-alert--m-inline--m-info__icon--Color: var(--pf-global--info-color--100);
-  --pf-c-alert--m-inline--m-info--before--BackgroundColor: var(--pf-global--info-color--100);
+  --pf-c-alert--m-inline--BackgroundColor: var(--pf-global--palette--cyan-50);
+
+  // Inline success
+  --pf-c-alert--m-inline--m-success--BackgroundColor: var(--pf-global--palette--green-50);
+
+  // Inline danger
+  --pf-c-alert--m-inline--m-danger--BackgroundColor: var(--pf-global--palette--red-50);
+
+  // Inline warning
+  --pf-c-alert--m-inline--m-warning--BackgroundColor: var(--pf-global--palette--gold-50);
+
+  // Inline info
+  --pf-c-alert--m-inline--m-info--BackgroundColor: var(--pf-global--palette--blue-50);
 
   @include pf-t-light; // This component always needs to be light
 
   position: relative;
   display: grid;
+  padding: var(--pf-c-alert--PaddingTop) var(--pf-c-alert--PaddingRight) var(--pf-c-alert--PaddingBottom) var(--pf-c-alert--PaddingLeft);
+  background-color: var(--pf-c-alert--BackgroundColor);
+  box-shadow: var(--pf-c-alert--BoxShadow);
   grid-template-columns: var(--pf-c-alert--grid-template-columns);
   grid-template-rows: var(--pf-c-alert--grid-template-rows);
   grid-template-areas:
     "icon title action"
-    "icon content content";
-  background-color: var(--pf-c-alert--BackgroundColor);
-  box-shadow: var(--pf-c-alert--BoxShadow);
-
+    ". content content"
```

Comment:
Icon should span the rows. If icon font-size is increased, spanning all rows will prevent vertical misalignment.

```
"icon title action"
"icon content content"
"icon actiongroup actiongroup"
```
**Before**
![Screen Shot 2020-04-07 at 10 05 41 AM](https://user-images.githubusercontent.com/5385435/78678952-9e652b00-78b7-11ea-8b6e-4e30e1b84355.png)

**After**
![Screen Shot 2020-04-07 at 10 05 24 AM](https://user-images.githubusercontent.com/5385435/78678990-a9b85680-78b7-11ea-90d5-58456ac3e036.png)